gitops/kcp/gateway-bindings/compute-apibinding.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIBinding
+metadata:
+  name: cluster-workspace
+spec:
+  reference:
+    workspace:
+      exportName: kubernetes
+      # TODO: the path needs to match the location in the cluster
```

Comment:
Am I understanding correctly that we should have a kustomization.yaml applied on top in the gitops SRE repository that would overwrite the path with that environment value?

Reply:
yes this is correct. The kustomize patch is available under Prerequisites in docs/gateway.md